### ISSUE TITLE
Main README working code

### ISSUE THREAD

**Issue by psytron** [jakearchibald/idb#248]
Would be awesome if the example code on main page would be working code when users cut and paste. Right now the (...) ... in examples means users must do more digging to get examples working. Working code would speed up learning and adoption for new joiners.

**Comment by jakearchibald** [comment]
Those are there intentionally because the examples are incomplete. The ellipsis needs to be replaced with something.

There are other examples on the page that are complete.

Would a "sandbox" be useful? As in, a page that exposes the API on the global and sets up a couple of databases, so folks could play with them in the console?

**Comment by psytron** [comment]
Yeah, sandbox would be sweet too.   Your examples on main page are perfect, if they were working code it would be very fast to get started and see the product in action quickly.
